### ISSUE TITLE
[API] Remove "V4" version string from our endpoints

### ISSUE THREAD

**Issue by CarlosRS9** [wazuh/wazuh#5629]
Hello team,

Before releasing the new API we need to remove any version (`v4`) string from our endpoints. 

## Description

We must check where we are including the `V4` string and update it. This version strings is included in the following places:

- `spec.yaml`: `v4` is included in the server's url section.
- `common.yaml` has a `version` variable used in every integration test.

Review if this version string is used in any other place not listed and update it.

Once all changes are made we must ensure everything still works as expected both manually and with swagger. Finally, `changes.md` should be updated accordingly.

##  Tasks
* [x] Remove the `v4` string.
* [x] Check manually that everything is still working as expected.
* [x] Run integration tests to ensure the changes didn't break anything.
* [x] Update `changes.md`

**Comment by mcarmona99** [comment]
After removing "v4" string, Wazuh-API was not able to start.
I found the next error after debugging and watching logs.

`connexion.exceptions.ConnexionException: aiohttp doesn't allow to set empty base_path ('/'), use non-empty instead, e.g /api`

This message says we cannot remove the "v4" string. The alternatives we have here are changing the string "v4" to another one (e.g. "api" or "wazuh") or leaving the "v4" string. 

To sum up, base_path ("/v4") cannot be /.
